tests/nn/data_parallel/test_sharded_ddp_pytorch_parity.py

```diff
@@ -54,6 +54,7 @@ def run_ddp_parity(
     fp16_reduction,
     clip_grad_norm,
     amp,
+    manual_reduction,
```

Comment:
manual reduction becomes a case like the others, having it as an insider case switch proved not to be foolproof, keeping up with the test matrix approach